src/Http/RequestManager.php

```diff
@@ -0,0 +1,126 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\Http;
+
+use Http\Client\Common\Plugin\AuthenticationPlugin;
+use Http\Client\Common\PluginClient;
+use Http\Client\HttpClient;
+use Http\Discovery\HttpClientDiscovery;
+use Http\Discovery\MessageFactoryDiscovery;
+use Http\Message\MessageFactory;
+use Lmc\Matej\Http\Plugin\ExceptionPlugin;
+use Lmc\Matej\Model\Request;
+use Lmc\Matej\Model\Response;
+use Psr\Http\Message\RequestInterface;
+
+/**
+ * Encapsulates HTTP layer, ie. request/response handling.
+ * This class should typically not be used directly - its supposed to be called internally from `Matej` class.
```

Comment:
"This class should not be typically used directly."

Reply:
Adjusted, thanks